SharkCage/CageManager/CageLabeler.cpp

```diff
@@ -16,11 +16,15 @@ using namespace Gdiplus;
 #include <optional>
 #include <cstring>
 
-static bool DisplayTokenInCageWindow(HWND *hwnd);
+static bool DisplayTokenInCageWindow(HDC hdc);
 static bool GetBottomFromMonitor(int &monitor_bottom);
 static bool ShowExitButton(HWND &hwnd);
 static bool ShowConfigMetadata(HWND &hwnd);
 
+static HWND hwnd_lb;
```

Comment:
What means `hwnd_lb`? Can't figure out what the lb means 😅

Reply:
the hwnd of the labeler (lb), do you prefer `hwnd_lbl` or `hwnd_labeler`?